test/docker/nginx-plus/deb/Dockerfile

```diff
@@ -43,13 +48,15 @@ RUN --mount=type=secret,id=nginx-crt,dst=nginx-repo.crt \
 # Uncomment individual modules if necessary
 # Use versioned packages over defaults to specify a release
     && nginxPackages=" \
-        nginx-plus \
```

Comment:
did you mean to remove `nginx-plus`

Reply:
yeah that was intentional. When you install app-protect it also installs NGINX Plus.